mono/mini/simd-intrinsics-netcore.c

```diff
@@ -721,14 +721,14 @@ emit_x86_intrinsics (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature
 			EMIT_NEW_ICONST (cfg, ins, supported ? 1 : 0);
 			ins->type = STACK_I4;
 			return ins;
-		case SN_MultiplyNoFlags:
-			// TODO: implement using _mulx_u32/u64:
-			// ulong MultiplyNoFlags(ulong left, ulong right)
-			// ulong MultiplyNoFlags(ulong left, ulong right, ulong* low) => MultiplyNoFlags(left, right, low);
-			// uint MultiplyNoFlags(uint left, uint right)
-			// uint MultiplyNoFlags(uint left, uint right, uint* low)
-			return NULL;
-		case SN_ZeroHighBits:
+		//case SN_MultiplyNoFlags:
+			//// TODO: implement using _mulx_u32/u64:
+			//// ulong MultiplyNoFlags(ulong left, ulong right)
+			//// ulong MultiplyNoFlags(ulong left, ulong right, ulong* low) => MultiplyNoFlags(left, right, low);
+			//// uint MultiplyNoFlags(uint left, uint right)
+			//// uint MultiplyNoFlags(uint left, uint right, uint* low)
+			//return NULL;
+		//case SN_ZeroHighBits:
```

Comment:
One line too much of comment?

Reply:
you mean `////` - it's for when I(someone) will uncomment the whole block - the comments will remain comments :)

Reply:
had to comment SN_MultiplyNoFlags for LLVM AOT mode (it crashed with vred != -1 assert)

Reply:
You commented out `case SN_ZeroHighBits` too.

Reply:
oops 🙂